e2e/app/page/criteria-search-page.ts

```diff
@@ -1,163 +1,53 @@
 import { Page } from 'puppeteer';
-import ReviewCriteriaSidebar from 'app/component/review-criteria-sidebar';
-import SelectMenu from 'app/component/select-menu';
 import Table from 'app/component/table';
-import Button from 'app/element/button';
-import ClrIconLink from 'app/element/clr-icon-link';
 import Textbox from 'app/element/textbox';
-import { waitForNumericalString, waitWhileLoading } from 'utils/waits-utils';
-import { LinkText } from 'app/text-labels';
+import { waitWhileLoading } from 'utils/waits-utils';
 import { getPropValue } from 'utils/element-utils';
 import AuthenticatedPage from './authenticated-page';
 
-export enum PhysicalMeasurementsCriteria {
-  BloodPressure = 'Blood Pressure',
-  BMI = 'BMI',
-  BPHypotensive = 'Hypotensive (Systolic <= 90 / Diastolic <= 60)', // BP prefix for Blood Pressure
-  BPNormal = 'Normal (Systolic <= 120 / Diastolic <= 80)', // Blood Pressure
-  BPPrehypertensive = 'Pre-Hypertensive (Systolic 121-139 / Diastolic 81-89)', // Blood Pressure
-  BPHypertensive = 'Hypertensive (Systolic >= 140 / Diastolic >= 90)', // Blood Pressure
-  HeartRate = 'Heart Rate',
-  Height = 'Height',
-  HipCircumference = 'Hip Circumference',
-  HRNoIrregularity = 'No-irregularity detected', // Heart Rhythm Status
-  HRIrregularity = 'Irregularity detected', // Heart Rhythm Status
-  PregnantEnrollment = 'Pregnant at enrollment',
-  WaistCircumference = 'Waist Circumference',
-  Weight = 'Weight',
-  WheelChairUser = 'Wheelchair user at enrollment'
-}
-
-export enum Visits {
-  AmbulanceVisit = 'Ambulance Visit',
-  AmbulatoryClinicCenter = 'Ambulatory Clinic/Center',
-  AmbulatoryRehabilitationVisit = 'Ambulatory Rehabilitation Visit',
-  EmergencyRoomVisit = 'Emergency Room Visit',
-  EmergencyRoomAndInpatientVisit = 'Emergency Room and Inpatient Visit',
-  HomeVisit = 'Home Visit',
-  InpatientVisit = 'Inpatient Visit',
-  LaboratoryVisit = 'Laboratory Visit',
-  NonhospitalInstitutionVisit = 'Non-hospital institution Visit',
-  OfficeVisit = 'Office Visit',
-  OutpatientVisit = 'Outpatient Visit',
-  PharmacyVisit = 'Pharmacy visit'
-}
-
-export enum FilterSign {
-  Any = 'Any',
-  AnyValue = 'Any value',
-  Equals = 'Equals',
-  GreaterThanOrEqualTo = 'Greater Than or Equal To',
-  LessThanOrEqualTo = 'Less Than or Equal To',
-  Between = 'Between'
-}
-
 export default class CriteriaSearchPage extends AuthenticatedPage {
```

Comment:
To be removed in follow up PR because this page function should be part of `cohort-participants-group.ts` and `review-criteria-sidebar.ts`.